generators/client/templates/src/main/webapp/app/services/profiles/_active-profiles.service.js

```diff
@@ -0,0 +1,69 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('<%=angularAppName%>')
+        .service('ActiveProfiles', ActiveProfiles);
+
+    ActiveProfiles.$inject = ['$q','$resource'];
+
+    function ActiveProfiles ($q, $resource) {
+
+        var _activeProfiles,_ribbonEnv,_inProduction,_response;
+
+	    var service = {
+	    	activeProfiles: activeProfiles,	    		
+	    	ribbonEnv: ribbonEnv,
+	        inProduction: inProduction,
+	        fetch:fetch
```

Comment:
do not expose fetch, always use `activeProfiles` so that user doesnt have to know if its fetched from server or from service cache